server/ladders-local.ts

```diff
@@ -292,11 +288,13 @@ class LadderStore {
 
 			let reasons = '' + (Math.round(p1newElo) - Math.round(p1elo)) + ' for ' + (p1score > 0.9 ? 'winning' : (p1score < 0.1 ? 'losing' : 'tying'));
 			if (reasons.charAt(0) !== '-') reasons = '+' + reasons;
-			room.addRaw(Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; <strong>${Math.round(p1newElo)}</strong><br />(${reasons})`);
+			room.addRaw(Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; ` +
+			`<strong>${Math.round(p1newElo)}</strong><br />(${reasons})`);
 
 			reasons = '' + (Math.round(p2newElo) - Math.round(p2elo)) + ' for ' + (p2score > 0.9 ? 'winning' : (p2score < 0.1 ? 'losing' : 'tying'));
 			if (reasons.charAt(0) !== '-') reasons = '+' + reasons;
-			room.addRaw(Chat.html`${p2name}'s rating: ${Math.round(p2elo)} &rarr; <strong>${Math.round(p2newElo)}</strong><br />(${reasons})`);
+			room.addRaw(Chat.html`${p2name}'s rating: ${Math.round(p2elo)} &rarr; ` +
```

Comment:
```suggestion
			room.addRaw(
				Chat.html`${p2name}'s rating: ${Math.round(p2elo)} &rarr; <strong>${Math.round(p2newElo)}</strong><br />(${reasons})`
			);
```

Same issue. Your second line isn't being sanitized.